sections/semantics-scriptings.include

```diff
@@ -2035,3 +2035,316 @@
+
+      Since only existing table elements takes part in the <a>table model</a>,
+      with other elements moved outside the <{table}>, creating a custom element that is part of a <{table}>
+      can only be done by customizing a built-in table element.
+  </div>
+
+  A <a href="#customized-built-in-element">customized built-in element</a> inherits the semantics of the element that it extends.</p>
+
+  A <a>customized built-in element</a> has the same requirements for attributes as the element it extends.
+  To add custom attribute-based behavior, use [=custom data attributes|data-*=] attributes.</p>
+
+  <p class="note">A <a>customized built-in element</a>'s <a>is value</a>
+   is saved on the element when it is [=create an element|created=].
+  This means changing the value of the <{global/is}> attribute after the element is
+  [=create an element|created=] does not change the element's behaviour.</p>
+
+</section>
+<section>
+<h4 id="requirement-for-all-custom-elements">Requirements for all custom elements</h4>
+
+  A <dfn id="valid-custom-element-name">valid custom element name</dfn> is a sequence of characters
+  <var>name</var> that matches the <a>PotentialCustomElementName</a> production,
+  and is not one of the following:</p>
+
+    <ul class="brief">
+      <li><code>annotation-xml</code></li>
+      <li><code>color-profile</code></li>
+      <li><code>font-face</code></li>
+      <li><code>font-face-src</code></li>
+      <li><code>font-face-uri</code></li>
+      <li><code>font-face-format</code></li>
+      <li><code>font-face-name</code></li>
+      <li><code>missing-glyph</code></li>
+    </ul>
+
+    <p class="note">These are all hyphen-containing element names from the <a>applicable specifications</a>: <cite>SVG</cite> and <cite>MathML</cite>. [[SVG11]] [[MATHML]]</p>
+  </li>
+</ul>
+
+  The <dfn><code>PotentialCustomElementName</code></dfn> production is as follows:
+<dl>
+  <dt><a>PotentialCustomElementName</a> ::=</code></dt>
+
+  <dd><code>[a-z] (<a href="#prod-pcenchar">PCENChar</a>)* '-' (<a href="#prod-pcenchar">PCENChar</a>)*</code></dd>
+
+  <dt><code><dfn id="prod-pcenchar">PCENChar</dfn> ::=</code></dt>
+
+  <dd><code>"-" | "." | [0-9] | "_" | [a-z] | #xB7 | [#xC0-#xD6] | [#xD8-#xF6] | [#xF8-#x37D] | [#x37F-#x1FFF] | [#x200C-#x200D] | [#x203F-#x2040] | [#x2070-#x218F] | [#x2C00-#x2FEF] | [#x3001-#xD7FF] | [#xF900-#xFDCF] | [#xFDF0-#xFFFD] | [#x10000-#xEFFFF]</code></dd>
+</dl>
+
+<p>This uses the <a>EBNF notation</a> from the <cite>XML</cite> specification. [[!XML]]</p>
+
+<div class="note">
+  <p>These requirements ensure a number of goals for <a>valid custom element names</a>:</p>
+
+  <ul>
+    <li>
+      <p>They start with a <a>lowercase ASCII letter</a>, ensuring that the HTML parser will treat them as tags instead of as text.</p>
+    </li>
+
+    <li>
+      <p>They do not contain any <a>uppercase ASCII letters</a>, so user agent can always treat HTML elements ASCII-case-insensitively.</p>
```

Comment:
I think it should be either “so **the** user agent can” or “so user agent**s** can”.